tickets.md

```diff
@@ -63,23 +63,23 @@ be deducted at checkout.
   <thead>
     <tr>
       <th>Ticket Type</th>
-      <th>Upfront cost (incl. eventbrite fee)</th>
+      <th>Upfront cost (excl. eventbrite fee)</th>
       <th>At checkout</th>
       <th>Total</th>
     </tr>
   </thead>
   <tbody>
     <tr>
-      <td>Twin room (2 beds) <br> sharing with one other person</td>
```

Comment:
(leaving it here because I can't leave it above - thanks github)
please change lines 57-60:

> The below prices are the final amount due before the event - they will likely be reduced by 
[sponsorships][sponsorship_page], which will be equally applied to all tickets. 
The price you pay at initial registration on Eventbrite serves as a deposit. The rest will be paid by bank transfer.

Reply:
please change line 88
`	<p><span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span> Note, <strong>the deposit is non-refundable.</strong> <br /><br />If after you bought the ticket you realise you can't make it, you can sell it to someone else. Send us the name of the person that will be attending in your place.<br /><br /> <strong>You will be liable for paying the remaining amount to us before the event </strong>— you will have a room booked in your name.</p>`